joint_pds:
  "0":
    kp: 30.0
    kd: 3.0
  "1":
    kp: 30.0
    kd: 3.0
  "2":
    kp: 30.0
    kd: 3.0
  "3":
    kp: 30.0
    kd: 3.0
  "4":
    kp: 30.0
    kd: 3.0
  "5":
    kp: 30.0
    kd: 3.0
  "6":
    kp: 30.0
    kd: 3.0
  "7":
    kp: 30.0
    kd: 3.0
  "8":
    kp: 30.0
    kd: 3.0
  "9":
    kp: 30.0
    kd: 3.0
